tests/product/test_installer.py

```diff
@@ -42,30 +42,12 @@ def tearDown(self):
         self.centos_container.tear_down()
 
     @attr('smoketest')
-    def test_online_installer(self):
-        self.pa_installer._build_installer_in_docker(self.centos_container,
-                                                     online_installer=True,
-                                                     unique=True)
+    def test_installer(self):
```

Comment:
`prestoadmin_installer` takes it from `dist` directory. `dist` directory is a place where all `make dist dist-online dist-offline docker-dist docker-dist-online docker-distoffline` put result artifact. Previously `prestoadmin_installer` took that from `tmp/installer` which was akward to me.